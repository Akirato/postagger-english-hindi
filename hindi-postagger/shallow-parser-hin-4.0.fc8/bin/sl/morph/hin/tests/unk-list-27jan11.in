sabakA
wahasa
nahasa
valavalA
pe
SAxi
102.3x
rASixa
bombe
pUne
usI
lagjarI
nqwyAMganA
PAibrOYyadislepsiyA_OYrIPikeMsa_progresivA
PZa√®lAI_ovarsa
rielitI
bAMta
dAMta
DUMDZa
kAMpa
maMgA
muskurA
saMBala
saMBAla
Wiraka
baMta
DUMDa
kIrA
suMGa
SvaSrU
vijibilitI
xilavA
