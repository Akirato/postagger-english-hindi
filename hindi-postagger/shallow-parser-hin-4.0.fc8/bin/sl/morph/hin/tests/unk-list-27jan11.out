1	wahasa		<fs af='wahasa,unk,,,,,,'>
2	valavalA		<fs af='valavalA,n,m,sg,3,d,0,0'>|<fs af='valavalA,n,m,pl,3,d,0,0'>|<fs af='valavalA,n,m,sg,3,o,0,0'>|<fs af='valavalA,n,m,pl,3,o,0,0'>
3	SAxi		<fs af='SAxi,unk,,,,,,'>
4	rASixa		<fs af='rASixa,adj,any,any,,any,,'>|<fs af='rASixa,n,m,sg,3,d,0,0'>|<fs af='rASixa,n,m,pl,3,d,0,0'>|<fs af='rASixa,n,m,sg,3,o,0,0'>|<fs af='rASixa,n,m,pl,3,o,0,0'>
5	pUne		<fs af='pUne,n,m,sg,3,d,0,0'>|<fs af='pUne,n,m,pl,3,d,0,0'>|<fs af='pUne,n,m,sg,3,o,0,0'>|<fs af='pUne,n,m,pl,3,o,0,0'>
6	lagjarI		<fs af='lagjarI,adj,any,any,,any,,'>
7	PAibrOYyadislepsiyA_OYrIPikeMsa_progresivA		<fs af='PAibrOYyadislepsiyA_OYrIPikeMsa_progresivA,unk,,,,,,'>
8	rielitI		<fs af='rielitI,n,f,sg,3,d,0,0'>|<fs af='rielitI,n,f,pl,3,d,0,0'>|<fs af='rielitI,n,f,sg,3,o,0,0'>|<fs af='rielitI,n,f,pl,3,o,0,0'>
9	dAMta		<fs af='dAMta,v,any,any,any,,0,0'>
10	kAMpa		<fs af='kAMpa,v,any,any,any,,0,0'>
11	muskurA		<fs af='muskurA,v,any,any,any,,0,0'>
12	saMBAla		<fs af='saMBAla,v,any,any,any,,0,0'>
13	baMta		<fs af='baMta,v,any,any,any,,0,0'>
14	kIrA		<fs af='kIrA,v,any,any,any,,0,0'>
15	SvaSrU		<fs af='SvaSrU,n,m,sg,3,d,0,0'>|<fs af='SvaSrU,n,m,pl,3,d,0,0'>|<fs af='SvaSrU,n,m,sg,3,o,0,0'>|<fs af='SvaSrU,n,m,pl,3,o,0,0'>
16	xilavA		<fs af='xilavA,v,any,any,any,,0,0'>
